test/integration/framework/util.go

```diff
@@ -62,25 +62,6 @@ func DeleteNamespaceOrDie(c clientset.Interface, ns *v1.Namespace, t *testing.T)
 	}
 }
 
-// GetReadySchedulableNodes addresses the common use case of getting nodes you can do work on.
-// 1) Needs to be schedulable.
-// 2) Needs to be ready.
-// If EITHER 1 or 2 is not true, most tests will want to ignore the node entirely.
-// If there are no nodes that are both ready and schedulable, this will return an error.
-func GetReadySchedulableNodes(c clientset.Interface) (nodes *v1.NodeList, err error) {
-	nodes, err = checkWaitListSchedulableNodes(c)
-	if err != nil {
-		return nil, fmt.Errorf("listing schedulable nodes error: %s", err)
-	}
-	Filter(nodes, func(node v1.Node) bool {
-		return IsNodeSchedulable(&node) && isNodeUntainted(&node)
```

Comment:
lint is now complaining that isNodeUntainted is unused

Reply:
Removed.